### ISSUE TITLE
For firefox browser on android, I can not paste the copied text to the blank field

### ISSUE THREAD

**Issue by zhengger** [firasdib/Regex101#1077]
## Bug Description
I copied some text in the browser and then I switched to the regex101.com page to try to paste it for testing. But there is no "PASTE" command to select when I long-pressed the blank field. There are only "COPY" "SELECT ALL" and other options.
BTW, for chrome android edition, it works well and I can paste text to the blank field normally. 

fireforx for android: 62.0b17
android: MIUI10.1.1.0

**Comment by firasdib** [comment]
Seems to be a bug with codemirror. Not much I can do as of right now, I believe. I'll look into it.

**Comment by firasdib** [comment]
I have updated codemirror locally, which mentions a potential fix for your issue. This will be released in the coming weeks.

**Comment by firasdib** [comment]
The website has been updated. Can you check if this is still an issue?

**Comment by firasdib** [comment]
Reopen if this is still an issue

**Comment by zhengger** [comment]
Dear Firasdib!
I try It and the PASTE fhunction works on android's firefox now. 
And I find another possible cause which maybe be responsible partly for this issue. [a firefox add-on](https://addons.mozilla.org/en-US/firefox/addon/tap-translate/?src=search) which I have installed  will disable the PASTE function in the blank field at the first time the page loading finished. After I uninstalled the add-on the PASTE function worked very well.
Thanks for your immidiate update, Sir!
